Tests/Tests/UIImage+StripeTests.m

```diff
@@ -0,0 +1,42 @@
+//
+//  UIImage+StripeTests.m
+//  Stripe
+//
+//  Created by Brian Dorfman on 4/25/17.
+//  Copyright © 2017 Stripe, Inc. All rights reserved.
+//
+
+#import <XCTest/XCTest.h>
+
+#import "UIImage+Stripe.h"
+#import "STPImageLibrary+Private.h"
+
+@interface UIImage_StripeTests : XCTestCase
+
+@end
+
+@implementation UIImage_StripeTests
+
+- (void)testJpegImageResizing {
+    // Strategy is to grab an image from our bundle and resize it to
+    // both smaller and bigger size than it already is and make sure we get
+    // what we expect
+
+    UIImage *testImage = [STPImageLibrary safeImageNamed:@"stp_shipping_form@3x.png"
+                                     templateIfAvailable:NO];
+
```

Comment:
to make sure we're exercising the resize logic, we should probably start with an assert on the original size:
```
NSData *data = UIImageJPEGRepresentation(testImage, 0.5);
XCTAssertTrue(data.length > 50000);
```